common/fix_missing_lef.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+from os import path
+import argparse
+import subprocess
+
+parser = argparse.ArgumentParser(
+    description='This file fixes the missing information from the LEF files of the cells')
+
+parser.add_argument('--skywater_path', '-sp',required=True,
+                    help='skywater path')
+
+parser.add_argument('--modify_hs_ms_diodes', '-mhmd',action='store_true', default=False,
+                help="modifies the hs and ms diode information.")
+
+parser.add_argument('--modify_vnb_vpb', '-mvv',action='store_true', default=False,
+                help="modifies the vnb and vpb of all cells in all libraries.")
+
+
+args = parser.parse_args()
+skywater_path = args.skywater_path
+modify_hs_ms_diodes = args.modify_hs_ms_diodes
+modify_vnb_vpb = args.modify_vnb_vpb
+
+libraries = ['sky130_fd_sc_hd','sky130_fd_sc_hdll','sky130_fd_sc_hs','sky130_fd_sc_ms','sky130_fd_sc_ls']
+
+if modify_vnb_vpb:
+    for lib in libraries:
+        revert_modify_vpb_result_cmd = "sed -i 's/PIN VPB\\n    DIRECTION INOUT ;\\n    USE POWER ;/PIN VPB/g' {lef} ".format(
```

Comment:
Every one of these fixes should be referencing a bug on the google/skywater-pdk GitHub repository.

Reply:
@mithro: For the vnb and vpb pins, I communicated it with Tim Edwards, and as he said they will be updated in a day or two, so I felt there is no need to open an issue for this. However, I can open one just for the sake of traceability.

As for the ms and hs diodes, this has been a long-standing issue, for a couple of months now: https://github.com/google/skywater-pdk/issues/141

I'm sure they will eventually be fixed in the original PDK source, however, these are just an "until-then" solutions.

Reply:
Couple of things;
 * If there is no github issue tracking something, then it doesn't exist.
 * Adding a simple comment like "This can be removed when GitHub issue XXXX is fixed." is perfectly fine.

Reply:
Roger that.

Reply:
I added the comment: https://github.com/RTimothyEdwards/open_pdks/pull/32/commits/fcedadef081398e7742a8ddf23d2db9d52965427
And opened a bug report to track the vnb and vpb issue: https://github.com/google/skywater-pdk/issues/172